lib/veritech-server/src/handlers.rs

```diff
@@ -150,10 +150,8 @@ async fn action_run_request_task(
     state: AppState,
     cyclone_request: ActionRunRequest,
     reply_mailbox: Subject,
-) {
```

Comment:
These were infallible wrappers. If you want to fail, I'd delete all four wrappers and just call the inner function directly.